### DISCUSSION TITLE
Welcome to rbenv-for-windows Discussions!

### DISCUSSION THREAD

**Discussion by ccmywish** [ccmywish/rbenv-for-windows#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SKOLIA0** [comment]
Hi. I'm from Russia. The confrontation between the United States and Russia forces us to look for new methods of work. One of the highlights is the transition of REDMINE development from UBUNTU to WINDOWS.
I am grateful to the author @ccmywish for such a necessary idea and work as porting RBENV to WINDOWS.
This instruction is preparatory to the installation of the project, "written on the knee", very quickly and for myself. 
Instructions in Russian.
https://github.com/SKOLIA0/rbenv-for-windows/blob/main/README_RU.md

**Reply by ccmywish** [reply]
Windows is from the United States, and Ubuntu seems to embrace open source more.

I'm curious why you choose Windows rather than Ubuntu?

**Reply by SKOLIA0** [reply]
OPENVPN works well in UBUNTU, but conflicts with other cross-country VPNs.  in Windows, several VPN (organizations and countries) are working stably
Old versions of Windows are cleaned from espionage using the program https://win10tweaker.pro/

**Reply by SKOLIA0** [reply]
Russia is working against openvpn
